appinventor/blocklyeditor/src/versioning.js

```diff
@@ -1955,7 +1955,13 @@ Blockly.Versioning.AllUpgradeMaps =
     2: "ai1CantDoUpgrade", // Just indicates we couldn't do upgrade even if we wanted to
 
     // RequestFocus was added
-    3: "noUpgrade"
+    3: "noUpgrade",
+
+    // TextChanged event, HintColor property, MoveCursorTo, MoveCursorToEnd and MoveCursorToFront methods were added.
+    4: [
+      Blockly.Versioning.changeMethodName("EmailPicker", "SetCursorAt", "MoveCursorTo"),
+      Blockly.Versioning.changeMethodName("EmailPicker", "SetCursorAtEnd", "MoveCursorToEnd"),
+      Blockly.Versioning.changeEventName("EmailPicker", "OnTextChanged", "TextChanged")]
```

Comment:
Just to confirm my understanding, the versioning here is to help with apps transitioning from Kodular, correct? Are we sure that the version numbers for each component are strictly greater than the ones used in Kodular?

Reply:
Sorry, they weren't. I now bumped up the version numbers, but it seems the build failed. What should I do?

Reply:
Updated.